jscomp/build_tests/cli_help/input.js

```diff
@@ -98,6 +98,19 @@ assert.match(
 // FIXME: Should be 0
 assert.equal(out.status, 1);
 
+// FIXME: Has the same problem with `rescript -w`
```

Comment:
Are you planning to fix this in a separate PR later?

Reply:
I'm thinking of doing it, but probably after v11 release. It'll require moving the arguments parsing from bsb to the `rescript.js`